src/common/Config.ts

```diff
@@ -14,9 +14,20 @@
  * limitations under the License.
  */
 
-import { isNode } from '../environment.js';
+interface Config {
+  WebSocketTransportClass: any;
```

Comment:
We have `ConnectionTransport` as a defined interface which we could use here rather than `any`

Reply:
the WebSocketTransportClass is just the Transport factory which should have a static create() method that return a ConnectionTransport instance.
so I need to define a factory interface for WebSocketTransportClass.